doc/guides/maintaining-V8.md

```diff
@@ -176,16 +176,20 @@ If the bug exists in any of the active V8 branches, we may need to get the fix b
 
 ### Backporting to Abandoned Branches
 
-Abandoned V8 branches are supported in the Node.js V8 repository. The fix needs
+Abandoned V8 branches are supported in the Node.js repository. The fix needs
 to be cherry-picked in the Node.js repository and V8-CI must test the change.
 
 * For each abandoned V8 branch corresponding to an LTS branch that is affected by the bug:
-    * Open a cherry-pick PR on nodejs/node targeting the appropriate *vY.x-staging* branch (e.g. *v6.x-staging* to fix an issue in V8-5.1).
+    * Checkout a branch off the appropriate *vY.x-staging* branch (e.g. *v6.x-staging* to fix an issue in V8 5.1).
+    * Cherry-pick the commit(s) from the V8 repository.
     * On Node.js < 9.0.0: Increase the patch level version in `v8-version.h`. This will not cause any problems with versioning because V8 will not publish other patches for this branch, so Node.js can effectively bump the patch version.
     * On Node.js >= 9.0.0: Increase the `v8_embedder_string` number in `common.gypi`.
     * In some cases the patch may require extra effort to merge in case V8 has changed substantially. For important issues we may be able to lean on the V8 team to get help with reimplementing the patch.
+    * Open a cherry-pick PR on nodejs/node targeting the *vY.x-staging* branch and notify the @nodejs/v8 team.
```

Comment:
I think this should have backticks around `nodejs/node`, to follow the rest of the doc.

Reply:
done